modules/apiHandler.py

```diff
@@ -232,4 +234,62 @@ def getCodeForcesLastSubmissionId(self, username):
             return data
         else:
             print("Could't get submission id. Please try this problem again later.")
-            return ""
\ No newline at end of file
+            return ""
+
+    # Online
+    def getSPOJSolveData(self, username):
+        """
+            This will be use in retrive already solved problem list of user in Spoj
+        """
+        start = 0 #We want to check from very last submission
+        self.submissions = []
+        previd = -1
+        currid = 0
+
+        # We assuming an user has at most 1000 page of submission on spoj (a single page has 20 submission history)
+        for page in range(1000):
+            flag = 0
```

Comment:
@sazidnur declare "flag" before the loop and be more specific about it's job. The name should answer what it holds. On a similar note, change the "start" variable's name.

Reply:
it's current submission id.. i redeclared and pushed again.